ci_framework/roles/hive/tasks/enable_root_login.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Enable root login
```

Comment:
we can do it better:
```YAML
- name: Activate passwordless root login
  register: sshd_config
  become: true
  ansible.builtin.copy:
    dest: /etc/ssh/sshd_config.d/root.conf
    content: |
      PermitRootLogin prohibit-password
- name: Restart sshd if needed
  when:
    - sshd_config is changed
  ansible.builtin.service:
    name: sshd
    state: restarted
```

Reply:
Awesome.

Sorry but since I have avoided using `root` user across the tasks.